app/src/pages/inside/projectSettingsPageContainer/content/notifications/notifications.jsx

```diff
@@ -185,7 +188,6 @@ export const Notifications = ({ setHeaderTitleNode }) => {
   };
 
   const readOnlyNotificationsEnableForm = !isAbleToEditNotificationsEnableForm();
```

Comment:
Please rename variable and corresponding function as now it is used not only to disable `EnableForm`.